stories/module-analytics-settings.stories.js

```diff
@@ -52,54 +51,7 @@ const completeModuleData = {
 	setupComplete: true,
 };
```

Comment:
Variables like `completeModuleData` and `incompleteModuleData` which are passed into the legacy settings component should no longer be necessary and can be removed.